l10n/en/firefox/features/fingerprinting.ftl

```diff
@@ -28,12 +30,14 @@ features-fingerprinting-fingerprinting-is-bad = Fingerprinting is bad for the we
 # $url_has (string) - link to https://www.cosic.esat.kuleuven.be/fpdetective/#paper
 # $url_steadily (string) - link to https://webtransparency.cs.princeton.edu/webcensus/
 # $url_increased (string) - link to https://sensor-js.xyz/
-features-fingerprinting-the-practice-of = The practice of fingerprinting allows you to be <a {{ $url_tracked }}>tracked for months</a>, even when you clear your browser storage or use private browsing mode — disregarding clear indications from you that you don’t want to be tracked. Despite a near complete agreement between <a {{ $url_standards }}>standards</a> <a {{ $url_bodies }}>bodies</a> and <a {{ $url_browser }}>browser</a> <a {{ $url_vendors }}>vendors</a> that fingerprinting is <a {{ $url_harmful }}>harmful</a>, its use on the web <a {{ $url_has }}>has</a> <a {{ $url_steadily }}>steadily</a> <a {{ $url_increased }}>increased</a> over the past decade.
```

Comment:
This and other strings had double braces `{{ ... }}` that are still replaced by the variable when the string is rendered, so I didn't think it was a problem, but it seems like a side effect of the double braces is that the string ID itself gets displayed in Pontoon. 

So from now on we should be on the lookout for double braces in strings and make sure that doesn't happen. TIL!